docs/basics.md

```diff
@@ -239,6 +239,33 @@ task my_task
 
 Since `current_date` is overridden, the callback is never executed.
 
+:::note
+If you need to create a new configuration option based on the overridden one, use dynamic configuration syntax:
+
+```php
+
+set('dir_name', 'test');
+
+set('uses_overridden_dir_name', function () {
+    return '/path/to/' . get('dir_name');
+});
+
+set('uses_original_dir_name', '/path/to/' . get('dir_name'));
```

Comment:
Lets clarify what calling get during recipe initialization will give you original value.